src/dotty/tools/dotc/core/TypeApplications.scala

```diff
@@ -417,29 +414,11 @@ class TypeApplications(val self: Type) extends AnyVal {
 
   /** If this is the image of a type argument to type parameter `tparam`,
```

Comment:
This still refers to `tparam` even though `tparam` is not a parameter of this method anymore. I think the comment could be simplified to just ` If this is the image of a type argument for a type parameter, recover the type argument, otherwise NoType.`, although I'd appreciate it if you could explain the meaning of `image` in the documentation.